recipes/kmod/all/conanfile.py

```diff
@@ -38,8 +37,14 @@ class KModConan(ConanFile):
     }
 
     def configure(self):
-        del self.settings.compiler.libcxx
-        del self.settings.compiler.cppstd
+        try:
+           del self.settings.compiler.libcxx
```

Comment:
Is there an issue for a "safe delete" feature so we dont need this mess?

Reply:
https://github.com/conan-io/conan/issues/11792

Reply:
Unfortunately not, I asked about it here: https://github.com/conan-io/conan/issues/11801